건강 체크 엔드포인트 응답 타입 오류

## 버그 설명
건강 체크 엔드포인트(`/health`)에서 `status` 값이 문자열 대신 숫자로 반환되고 있습니다.

## 재현 방법
1. 테스트 실행: `python -m pytest tests/test_backend/test_routers/test_main.py -v`
2. 테스트 실패 확인

## 에러 로그
```
AssertionError: assert {'status': 1} == {'status': 'healthy'}
```

## 예상 동작
- 현재 동작: `{"status": 1}`
- 기대 동작: `{"status": "healthy"}`

## 우선순위
높음 - 기본 API 테스트 실패

## 관련 파일
- `backend/main.py`
- `tests/test_backend/test_routers/test_main.py`

--- a/backend/main.py
+++ b/backend/main.py
@@ -21,7 +21,7 @@ async def root():
 
 @app.get("/health")
 async def health_check():
-    return {"status": 1}
+    return {"status": "healthy"}
 
 if __name__ == "__main__":
     import uvicorn

